homeassistant/components/roku/__init__.py

```diff
@@ -56,14 +56,18 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
 
 def roku_exception_handler(
-    func: Callable[Concatenate[_T, _P], Awaitable[None]]  # type: ignore[misc]
+    func: Callable[Concatenate[_T, _P], Awaitable[None]],  # type: ignore[misc]
+    ignore_timeout: bool = False,
```

Comment:
Ah the joy of decorators in Python.

```py
@roku_exception_handler
```

is not the same as

```py
@roku_exception_handler()
```

In the second case, you need to return a function that will be used as a decorator.

Reply:
ah OK likely better to make a second helper for such then? since variable behavior?

Reply:
You can update all uses of this decorator to call it, with or without args

Reply:
ok hopefully the current version will work out after a few failed variations